content/10.methods.md

```diff
@@ -146,3 +146,11 @@ The table lists the 8 groups and the number of living people for each region tha
 Example names shows actual author names that received a high prediction for each region.
 Full information about which countries comprised each region can be found in the online dataset [`country_to_region.tsv`](https://github.com/greenelab/wiki-nationality-estimate/blob/master/data/country_to_region.tsv).
 {#tbl:example_names}
+
+### Affiliation Analysis
+
+Along with the corresponding author names, we collected their affiliations recorded in each publication for this analysis.
+During the honoree curation process, if an honoree was listed with their affiliation at the time, we recorded this affiliation for analysis.
+Because we could not find affiliations for the 1997 and 1998 RECOMB keynote speakers' listed for these years, they were left blank.
+We used the [standard world geographical mapping](https://raw.githubusercontent.com/greenelab/iscb-diversity/b41b8fad157d0b878a2476cec998aba0643742bf/figs/2020-01-31_natural-earth-world-map.png) from the R package `rnaturalearth` to group the countries of affiliation in seven different regions: North America, Europe & Central Asia, East Asia & Pacific, Latin America & Caribbean, South Asia, Middle East & North Africa and Sub-Saharan Africa.
```

Comment:
I can't find anything online for "standard world geographical mapping". Tracking the data back its from the `region_wb` column of [`data/countries/world-map.tsv`](https://github.com/greenelab/iscb-diversity/blob/96ad84b0c58c3b3de301056f7546664f8bacb057/data/countries/world-map.tsv). Sounds like this is from Natural Earth. I'll look for some documentation of `region_wb` that we can point to... basically who makes it and via what criteria / what `region_wb` is defined as.

Reply:
I agree that we need to source the mapping.